lib/endpoints/class-wp-rest-controller.php

```diff
@@ -250,7 +250,6 @@ public function get_collection_params() {
 				'default'            => 10,
 				'minimum'            => 1,
 				'maximum'            => 100,
-				'sanitize_callback'  => 'absint',
```

Comment:
Ya sanitization happens before validation and there is a lot of things that depend on it from what I have seen.  Would there be an argument against doing something like `sanitize_callback => 'rest_intval',` , then somewhere `function rest_intval( $value, $request, $param ) { return intval( $value ); }`.  I like the white list approach but maybe use a sort of stop gap solution before white list is in core??